cyclops-ctrl/internal/cluster/k8sclient/client.go

```diff
@@ -772,6 +774,32 @@ func (k *KubernetesClient) mapPersistentVolumeClaims(group, version, kind, name,
 	}, nil
 }
 
+func (k *KubernetesClient) mapPersistentVolumes(group, version, kind, name, namespace string) (*dto.PersistentVolume, error) {
+	persistentvolume, err := k.clientset.CoreV1().PersistentVolumes().Get(context.Background(), name, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	capacity := ""
+	if persistentvolume.Spec.Capacity != nil && persistentvolume.Spec.Capacity.Storage() != nil {
+		capacity = persistentvolume.Spec.Capacity.Storage().String()
+	}
+
+	return &dto.PersistentVolume{
+		Group:                 group,
+		Version:               version,
+		Kind:                  kind,
+		Name:                  name,
+		Namespace:             namespace,
+		AccessModes:           persistentvolume.Spec.AccessModes,
+		PersistentVolumeClaim: persistentvolume.Spec.ClaimRef.Name,
```

Comment:
This can also produce a nil pointer dereference since `ClaimRef` is a pointer. Can you solve it the same way as capacity?

Reply:
fixed using "Retain" as a default policy

Reply:
fixed as suggested